### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by natalia-matash**
- [DEMO LINK](https://natalia-matash.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://natalia-matash.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Comment by natalia-matash** [comment]
I was faced  with a problem. Firtly, I didn't want to fix size of image, so I did this way, like in mockup:
.card__image {
  padding: 32px 19px 0;
  margin-bottom: 40px;
}
.card__img {
  max-width: 100%;
}
But, image didn't cover all block and I had 0.8px bottom backlash. I was trying object fit, coudn't help.
Same problem with max-height, height, weight.
So, I had fixed size of clock

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
button in hover state doesn't look like in mockup 
image

File: src/styles/blocks/cards.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.card {
```
Comment (SanyaBratashchuk) [inline src/styles/blocks/cards.css:1]: rename this file cards.css  => card.css

**Review by SanyaBratashchuk — APPROVED** [review]

**Comment by natalia-matash** [comment]
Done

вс, 8 янв. 2023 г. в 13:20, Sasha Bratashchuk ***@***.***>:

**Pull request closed without merging**